webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -81,6 +96,19 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [page, sortString]);
 
+  useEffect(() => {
+    let ignore = false;
```

Comment:
**nit**: @keita-determined as per discussion could you add a comment on why you implemented the useEffect this way, perhaps document the bug/issue this solves.